server/args.py

```diff
@@ -0,0 +1,18 @@
+# args.py
+import argparse
+
+
+def parse_args():
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument(
+        "--sqlite-file-path",
+        default="pastes.db",
+        help="The path to the SQLite database file.",
+    )
+
+    parser.add_argument("--host", default="localhost", help="add an IP adress")
```

Comment:
look at the below for better help text for host and port

https://github.com/SCE-Development/cleezy/blob/a37893f33252857e085bfbe06376b137129bf992/modules/args.py#L12

Reply:
Changed the help text based on the link, ty